src/views/dashboard/ChangeSecret/CardSummary.vue

```diff
@@ -116,12 +116,20 @@ export default {
 }
 </script>
 
-<style scoped>
+<style scoped lang="scss">
 .margin-top-10 {
   margin-top: 10px;
 }
 
 ::v-deep .content .summary-card {
   flex: 1;
+  width: 33%;
+  max-width: 33%;
+
+  .summary-header {
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+  }
 }
 </style>
```

Comment:
There is no known issue present, but here are some suggestions:

  * The usage of `export` statement with curly braces in JavaScript seems to be outdated. Instead, try using an asterisk or the colon (:) method like this: `export default`.
    ```javascript
    export default function() { ... };
```
  * Style components must use 'scoped' CSS rules instead of inline styles. Use ':v-deep'. Consider wrapping them in ::after pseudo element for consistency.
  * In SCSS files, it's best practice to write only one level deep. Try writing more than one line on each new-line so that they can be easily grouped and expanded without needing additional spaces.

In conclusion, you have provided almost complete syntax guidelines which need adjustments to align better and adhere to modern practices.